app/components/filter-sidebar/filter/template.hbs

```diff
@@ -1,46 +1,61 @@
-{{#if (eq this.type 'button')}}
-  <div class="btn-group btn-group--auto">
-    {{#each this.options as |opt|}}
-      <button
-        class="btn {{if (eq (get opt this.valuePath) this.selected) 'active'}}" type="button" {{action (optional this.on-change) (get opt this.valuePath)}}
+<div ...attributes>
+  {{#if (eq @type "button")}}
+    <div class="btn-group btn-group--auto">
+      {{#each @options as |opt|}}
+        <button
+          class="btn {{if (eq (get opt @valuePath) @selected) 'active'}}"
+          type="button"
+          {{on "click" (fn (optional @onChange) (get opt @valuePath))}}
+        >
+          {{get opt @labelPath}}
+        </button>
+        {{yield}}
+      {{/each}}
+    </div>
+  {{else if (eq @type "date")}}
+    <SyDatepicker
+      @value={{@selected}}
+      @onChange={{optional @onChange}}
+      @placeholder={{(or @placeholder)}}
```

Comment:
Does `@placeholder` behave weirdly when a falsy value is passed or why not just `{{@placeholder}}`?

Reply:
Shit, that's a debugging artifact.